aten/src/ATen/native/metal/MetalConvParams.h

```diff
@@ -32,7 +32,7 @@ struct Conv2DParams final {
   int64_t H; // input height
   int64_t W; // input width
   int64_t OC; // output channels
-  int64_t IC; // input channels
+  uint64_t IC; // input channels
```

Comment:
This seems questionable lol

Reply:
Do you think input channels can be negative? ;) (I could have casted a bunch of invocations, but it felt safer to do it here

Reply:
It's questionable because it's not symmetric with the other fields, and that makes it more risky if someone accidentally tries to use the unsigned field as if it were signed

Reply:
why change only IC? I would think it would make sense to change other fields? Is this to enable "-Wno-sign-compare"? It is weird that other fields are still signed.

Reply:
Ok, let me revert this one and just typecast one assert instead.